skimage/util/montage.py

```diff
@@ -108,3 +121,68 @@ def montage2d(arr_in, fill='mean', rescale_intensity=False, grid_shape=None):
     arr_out = arr_out.reshape(alpha_y * height, alpha_x * width)
 
     return arr_out
+
+
+def montage_rgb(arr_in, fill='mean', grid_shape=None, padding_width=0):
```

Comment:
On a second thought, I'd suggest to create `montage` function with `multichannel` argument, incorporate 2d and 3d (and, potentially, nD) cases into it, and deprecate `montage_2d`.

If we agree on this, I could take care of the implementation.